### PR TITLE
Update particle_class.py

### LINKED ISSUE #757: Refactor Particle.__init__ to improve readability
While looking back through code in `plasmapy.particles`, I noticed that [`Particle.__init__`](https://github.com/PlasmaPy/PlasmaPy/blob/063c678fcaa79926a6de394a8000cf5076cd8fe7/plasmapy/particles/particle_class.py#L276) is rather long and is trying to do a whole bunch of different things.  Unfortunately, this makes the code harder to read and refactor.  We should break up `Particle.__init__` into smaller private methods with descriptive names that take care of the implementation details.  Each method should only do one thing.  Ideally, `Particle.__init__` and these methods should read like a top-down narrative (also known as the stepdown rule).  I recently also compiled some slides that review how to [write clean scientific software](https://zenodo.org/record/3491142).  Thank you!

### PR DESCRIPTION
**Pull request by faryal-khan**
Closes #757 

- Added two new private methods in particle_class.py, namely __checkingAlias__ and __setElementProperties__ in order to improve __init__'s readability and to make refactoring more convenient (had trouble running tests so currently submitting a draft)

- [ ] I have added a changelog entry for this pull request (please see
  `changelog/README.rst` for instructions, and if you need help with picking the PR type, ask!)
- [ ] If adding new functionality, I have added (passing) tests and
      docstrings. (Tests pop up at the bottom, in the checks box).
- [ ] I have fixed any new failing tests (if you're unsure why
  they're failing, ask!).

### CODE REVIEW

File: plasmapy/particles/particle_class.py [hunk 1]
```diff
@@ -273,51 +273,16 @@ class Particle:
 
     """
 
-    def __init__(
-            self,
+  
+   ##############
+
```
Comment (rocco8773) [inline plasmapy/particles/particle_class.py]: Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)

File: plasmapy/particles/particle_class.py [hunk 2]
```diff
@@ -368,7 +333,13 @@ def __init__(
             if self.element and self._attributes['integer charge']:
                 categories.add('ion')
 
-            # Element properties
+##########################
+
```
Comment (rocco8773) [inline plasmapy/particles/particle_class.py]: Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)

File: plasmapy/particles/particle_class.py [hunk 3]
```diff
@@ -409,6 +380,70 @@ def __init__(
 
             categories.add(Element['category'])
 
+
+
+
+
+##########################
+
```
Comment (rocco8773) [inline plasmapy/particles/particle_class.py]: Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)

File: plasmapy/particles/particle_class.py [hunk 4]
```diff
@@ -429,6 +464,8 @@ def __init__(
 
         self.__name__ = self.__repr__()
 
+##########
+
```
Comment (rocco8773) [inline plasmapy/particles/particle_class.py]: Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)

File: plasmapy/particles/particle_class.py [hunk 5]
```diff
@@ -273,51 +273,16 @@ class Particle:
 
     """
 
-    def __init__(
-            self,
+  
+   ##############
+
+    def __checkingAlias__(self,
```
Comment (rocco8773) [inline plasmapy/particles/particle_class.py]: [PEP8: for methods we use lowercase with underscore spacing](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)

This method also does not reach the standard for a dunder, so a single leading underscore will suffice, i.e. ``__checkingAlias__ -> _checking_alias``

File: plasmapy/particles/particle_class.py [hunk 6]
```diff
@@ -368,7 +333,13 @@ def __init__(
             if self.element and self._attributes['integer charge']:
                 categories.add('ion')
 
-            # Element properties
+##########################
+
+def __setElementProperties__( self,
```
Comment (rocco8773) [inline plasmapy/particles/particle_class.py]: [PEP8: for methods we use lowercase with underscore spacing](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)

This method also does not reach the standard for a dunder, so a single leading underscore will suffice, i.e. ``__setElementProperties__ -> _set_element_properties``

File: plasmapy/particles/particle_class.py [hunk 7]
```diff
@@ -409,6 +380,70 @@ def __init__(
 
             categories.add(Element['category'])
 
+
+
+
+
+##########################
+
+def __init__(
```
Comment (rocco8773) [inline plasmapy/particles/particle_class.py]: `__init__` should follow the class docstrings and its indentation needs to be fixed

**Comment by namurphy** [comment]
Oh, another thought!  We do a whole bunch of test runs to make sure things are working with different operating systems and versions of Python.  I usually find the Travis CI test run to be least difficult to navigate.  It should be fifth from the bottom in the list of tests, and clicking on "Details" should get you to the test results.

**Comment by StanczakDominik** [comment]
We really do need that pre-commit config, don't we...

**Pull request closed without merging**